resolwe_bio/processes/workflows/bbduk_star.py

```diff
@@ -0,0 +1,713 @@
+"""STAR-based RNA-seq pipeline."""
+
+from resolwe.process import (
+    BooleanField,
+    Data,
+    DataField,
+    FloatField,
+    GroupField,
+    IntegerField,
+    ListField,
+    Process,
+    StringField,
+)
+from resolwe.process.models import Process as BioProcess
+
+
+class WorkflowSTAR(Process):
+    """STAR-based RNA-seq pipeline.
+
+    First, reads are preprocessed by __BBDuk__ which removes adapters, trims
+    reads for quality from the 3'-end, and discards reads that are too short
+    after trimming. Compared to similar tools, BBDuk is regarded for its
+    computational efficiency.  Next, preprocessed reads are aligned by __STAR__
+    aligner. At the time of implementation, STAR is considered a
+    state-of-the-art tool that consistently produces accurate results from
+    diverse sets of reads, and performs well even with default settings. Aligned
+    reads are then summarized to genes by STAR. For more information see
```

Comment:
> Aligned reads are then summarized to genes by STAR

STAR aligner counts and reports the number of aligned reads per gene while mapping.